### PR TITLE
CORS security feature

### PR DESCRIPTION
**Pull request by CalBesley**
added CORS to settings.py

### CODE REVIEW

File: core/settings.py [hunk 1]
```diff
@@ -66,9 +67,12 @@
 
     'home',
     'theme_pixel'
```
Comment (vaqwerty2) [inline core/settings.py:69]: Please add a comma here on line 68 after 'theme_pixel'

**Comment by vaqwerty2** [comment]
Please include django-cors-headers in the requirements file.

**Comment by vaqwerty2** [comment]
The version currently in the requirements file is in compatible, use this version: 4.6.0

**Comment by vaqwerty2** [comment]
CORS implementation looks to be okay now.
Screenshot 2024-12-08 at 14 03 00

**Review by s-bonnor — APPROVED** [review]
Looks good, will be merged in a little bit

**Pull request merged**